docs/general/NodeOperatorList.md

```diff
@@ -0,0 +1,71 @@
+# List of available Node Operators
+
+## Node Operator list
+| Sifchain Address | Discord | Other blockchains I run Nodes for | How long do I plan on running a node? | Why would I stop running my node? | What kind of emergency response plans do I have? |
+|:-----------------|:--------|:----------------------------------|:--------------------------------------|:----------------------------------|:-------------------------------------------------|
+sif14tm9600fx088jw55gypcwkwh04j34e9jp68t8r
```

Comment:
whats this all about?

Reply:
@utx0 I am going to upload all node validator info here for people to reference when delegating. Still very much a WIP right now.